README.md

```diff
@@ -19,32 +19,50 @@ We assume that you already have:
 
 Then, you can create an URS for your circuit in the following way:
 
-```rust,ignore
-use kimchi::{circuits::constraints, verifier::verify};
+```rust
+use std::sync::Arc;
+
+use kimchi::{circuits::{constraints::ConstraintSystem, polynomials::generic::testing::{create_circuit, fill_in_witness}, polynomial::COLUMNS}, prover_index::ProverIndex, proof::ProverProof, verifier::verify};
+use ark_ff::Zero;
 use mina_curves::pasta::{fp::Fp, vesta::{Affine, VestaParameters}, pallas::Affine as Other};
 use oracle::{
     constants::PlonkSpongeConstantsKimchi,
     sponge::{DefaultFqSponge, DefaultFrSponge},
 };
-use commitment_dlog::commitment::{b_poly_coefficients, ceil_log2, CommitmentCurve};
+use commitment_dlog::{commitment::{CommitmentCurve}, srs::{SRS, endos}};
+use groupmap::GroupMap;
+use array_init::array_init;
+
 
 type SpongeParams = PlonkSpongeConstantsKimchi;
 type BaseSponge = DefaultFqSponge<VestaParameters, SpongeParams>;
 type ScalarSponge = DefaultFrSponge<Fp, SpongeParams>;
 
 // compile the circuit
+let gates = create_circuit(0, 0);
+
 let fp_sponge_params = oracle::pasta::fp_kimchi::params();
-let cs = ConstraintSystem::<Fp>::create(gates, vec![], fp_sponge_params, public_size).unwrap();
+let public = vec![Fp::from(0); 0];
+let mut witness: [Vec<Fp>; COLUMNS] = array_init(|_| vec![Fp::zero(); gates.len()]);
+fill_in_witness(0, &mut witness, &public);
```

Comment:
idea we wouldn't use testing functions like `create_circuit` and `fill_in_witness` but I guess without the circuit writer PR in we don't have much choices. cc @imeckler